### PR TITLE
untested way of fixing nukeops being loneop every round

### PR DESCRIPTION
**Pull request by checkraisefold**
:cl: zennerx
fix: Mediocre attempt at fixing nukeops being loneop every round!
/:cl:

well guess what this is untested because I can't make my own server with over 20 people ready at once
wait shit the requiredp layers is 30!!!

### CODE REVIEW

**Comment by checkraisefold** [comment]
holy shit it compiled on the first try

**Comment by ghost** [comment]
+1

**Comment by Manly-Dwarf** [comment]
Did you test this?
smug_high-res

**Comment by 2Dollarbill** [comment]
didyatestthisshityetson?

**Comment by checkraisefold** [comment]
'well guess what this is untested because I can't make my own server with over 20 people ready at once
wait shit the requiredp layers is 30!!!' @2Dollarbill @fluxcapacitor1337

**Comment by Pyko1** [comment]
not modular also isn't it upstream issue?

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -21,13 +21,17 @@
...
 /datum/game_mode/nuclear/pre_setup()
 	var/n_agents = min(round(num_players() / 10), antag_candidates.len, agents_possible)
-	for(var/i = 0, i < n_agents, ++i)
-		var/datum/mind/new_op = pick_n_take(antag_candidates)
-		pre_nukeops += new_op
-		new_op.assigned_role = "Nuclear Operative"
-		new_op.special_role = "Nuclear Operative"
-		log_game("[new_op.key] (ckey) has been selected as a nuclear operative")
-	return TRUE
+	if(n_agents < required_enemies)
```
Comment (Carbonhell) [inline code/game/gamemodes/nuclear/nuclear.dm]: You mean >?

Comment (ghost) [reply code/game/gamemodes/nuclear/nuclear.dm]: shouldn't it be >= since 2 agents in 2 required_enemies should pass?

**Comment by Carbonhell** [comment]
Cant you modularize this by putting this proc in a hippie file and doing
if(n_agents < required_enemies)
    Return false
..()

**Comment by Manly-Dwarf** [comment]
No, you can't mirror procs in other files.

**Comment by Carbonhell** [comment]
...you can?? It is literally how we have been doing stuff for the past months lmao

File: code/game/gamemodes/nuclear/nuclear.dm (hunk 1 above)
Comment (checkraisefold) [reply code/game/gamemodes/nuclear/nuclear.dm]: < does the same thing but it's a shorter operator

**Comment by checkraisefold** [comment]
@Carbonhell what hippie file goddamnit

**Comment by Carbonhell** [comment]
a new one made by you, preferably with same tg path so it'd be
hippiestation/code/game/gamemodes/nuclear/nuclear.dm
if you need futher help come on discord coderbus,i'll be available in 9-10 hours from now

**Comment by checkraisefold** [comment]
goddamnit how am I supposed to merge it all into one commit on remote

**Comment by checkraisefold** [comment]
it's better without super() (..()) because it would ruin the modular part because of my admin logging

**Comment by Carbonhell** [comment]
Why did you mirror the whole file aaaa 
Pm me on discord anytime and ill drive you step by step on how to solve this

**Comment by checkraisefold** [comment]
since nobody cares about this i'm making a pr upstream

**Pull request closed without merging**